crates/libs/registry/src/bindings.rs

```diff
@@ -72,6 +72,7 @@ pub const REG_EXPAND_SZ: REG_VALUE_TYPE = 2u32;
 pub const REG_MULTI_SZ: REG_VALUE_TYPE = 7u32;
 pub type REG_OPEN_CREATE_OPTIONS = u32;
 pub const REG_OPTION_NON_VOLATILE: REG_OPEN_CREATE_OPTIONS = 0u32;
+pub const REG_OPTION_VOLATILE: REG_OPEN_CREATE_OPTIONS = 1u32;
```

Comment:
The `bindings.rs` file is generated by `tool_bindings` - you need to update that crate to update the bindings rather than just updating it here manually.